launcher/ui/MainWindow.cpp

```diff
@@ -2093,21 +2094,23 @@ void MainWindow::on_actionDeleteInstance_triggered()
 
     auto id = m_selectedInstance->id();
 
-    auto response =
-        CustomMessageBox::selectable(this, tr("CAREFUL!"),
-                                     tr("About to delete: %1\nThis may be permanent and will completely delete the instance.\n\nAre you sure?")
-                                         .arg(m_selectedInstance->name()),
-                                     QMessageBox::Warning, QMessageBox::Yes | QMessageBox::No, QMessageBox::No)
-            ->exec();
+    auto response = CustomMessageBox::selectable(this, tr("CAREFUL!"),
```

Comment:
```suggestion
    auto response = CustomMessageBox::selectable(this, tr("Confirm deletion"),
```

no need to yell for that xD

Reply:
This was the text before which I was trying to conform to. I get that it's a little shouty though. I'll make sure to change it everywhere else too.